### PR TITLE
Extend Users API

### PR DESCRIPTION
**Pull request by mohabusama**
+ Bank handlers
+ Address handlers
+ Phone handlers

### CODE REVIEW

File: identityservice/user/users_api.go [hunk 1]
```diff
@@ -117,81 +141,483 @@ func (api UsersAPI) usernamevalidateGet(w http.ResponseWriter, r *http.Request)
...
+		if _, ok := user.Phone[label]; ok {
+			// Do not allow creating existing phone label!
+			http.Error(w, http.StatusText(http.StatusConflict), http.StatusConflict)
+			return
+		}
+
+		user.Phone[label] = val
+	}
+
+	if err := userMgr.Save(user); err != nil {
```
Comment (robvanmieghem) [inline identityservice/user/users_api.go]: I would not save the entire user, can cause concurrency issues. I would add a method to the user db manager to save a phone number and nothing but that. Mongo can do so:
`collection.Update(bson.M{"username": username}, bson.M{"$set": bson.M{"emails": bson.M{"testkey": testdata}}})`

**Comment by robvanmieghem** [comment]
@mohabusama thanks.
I added a comment for the emails but goes for the other properties that have separate rest calls for the management of them as well.

File: identityservice/user/users_api.go [hunk 2]
```diff
@@ -117,81 +141,483 @@ func (api UsersAPI) usernamevalidateGet(w http.ResponseWriter, r *http.Request)
 	// w.Header.Set("key","value")
 }
 
+// Create new phonenumber
+// It is handler for POST /users/{username}/phonenumbers
+func (api UsersAPI) usernamephonenumbersPost(w http.ResponseWriter, r *http.Request) {
```
Comment (robvanmieghem) [inline identityservice/user/users_api.go]: Can basic phonenumber validation be done to be in line with the limitations as specified in the raml file?

Comment (mohabusama) [inline identityservice/user/users_api.go]: Done

**Pull request merged**